chapters/en/chapter8/4.mdx

```diff
@@ -185,7 +185,7 @@ trainer.train_dataset[0].keys()
 dict_keys(['attention_mask', 'hypothesis', 'idx', 'input_ids', 'label', 'premise'])
 ```
 
-Note that the keys that don't correspond to inputs accepted by the model will be automatically discarded, so here we will only keep `input_ids`, `attention_mask`, and `label` (which will be renamed `labels`). To double-check the model signature, you can print the class of your model, then go check its documentation:
+Note that the keys that don't correspond to inputs accepted by the model as some will be automatically discarded, so here we will only keep `input_ids`, `attention_mask`, and `label` (which will be renamed `labels`). To double-check the model signature, you can print the class of your model, then go check its documentation:
```

Comment:
The original text is actually correct, and adding "as some" doesn't quite capture the original meaning. Here's an alternative formulation that might be a bit clearer:

> In most cases, your model will only accept a subset of the keys (i.e. columns) that are present in the dataset. To prevent errors during training and inference, the `Trainer` will thus discard any key that isn't present in the model's forward pass. In our case, this means only `input_ids`, `attention_mask`, and `label` (which will be renamed `labels`) will remain.

Reply:
I see now, I misread it.

```suggestion
Note that the keys that don't correspond to inputs accepted by the model will be automatically discarded, so here we will only keep `input_ids`, `attention_mask`, and `label` (which will be renamed `labels`). To double-check the model signature, you can print the class of your model, then go check its documentation:
```

Reply:
I see now, I misread it.

```suggestion
Note that the keys that don't correspond to inputs accepted by the model will be automatically discarded, so here we will only keep `input_ids`, `attention_mask`, and `label` (which will be renamed `labels`). To double-check the model signature, you can print the class of your model, then go check its documentation:
```